data/ecosystems/a/aave.toml

```diff
@@ -1,9 +1,9 @@
 # Ecosystem Level Information
 title = "Aave"
 
-sub_ecosystems = ["Aavegotchi", "Bored Ghosts", "Newt (WeAreNewt)"]
+sub_ecosystems = ["Aavegotchi", "Bored Ghosts", "Newt (WeAreNewt)", "Aave-Chan Initiative"]
```

Comment:
You'll need to create an ecosystem for this new added sub_ecosystem because it doesn't currently exist in the taxonomy. This means you'll need to create a new toml file for ACI and then add the corresponding orgs / repos to it

Reply:
awesome thanks @marczeller !